src/main/java/org/cloudfoundry/promregator/lite/discovery/DiscoveryControllerV2.kt

```diff
@@ -23,6 +24,7 @@ class DiscoveryLabelV2(
         @get:JsonGetter("__meta_promregator_target_applicationId") val applicationId: String? = null,
         @get:JsonGetter("__meta_promregator_target_instanceNumber") val instanceNumber: String? = null,
         @get:JsonGetter("__meta_promregator_target_instanceId") val instanceId: String? = null,
+        @get:JsonGetter("__meta_promregator_target_api") val api: String? = null,
```

Comment:
Note the new `api` attribute allows us to return results across multiple CF regions/apis.